packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.tsx

```diff
@@ -51,31 +52,35 @@ export class SearchBox extends BaseComponent<ISearchBoxProps, ISearchBoxState> {
   public render() {
     let { labelText, className } = this.props;
     let { value, hasFocus, id } = this.state;
+    const isActive = styles.isActive;
+    const canClear = styles.canClear;
 
     return (
       <div
         ref={ this._resolveRef('_rootElement') }
-        className={ css('ms-SearchBox', className, {
+        className={ css('ms-SearchBox', className, styles.root, {
           'is-active': hasFocus,
```

Comment:
minor nit; instead of duplicating rules, can you just merge them together? and not use the consts? That way it is much more clear what is going on.

```tsx
['is-active' + styles.rootIsActive]: hasFocus
```